front/app/api/analysis_users/useAnalysisUserById.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import usersKeys from './keys';
+import { IAnalysisUser, AnalysisUsersKeys } from './types';
+
+const useAnalysisUserById = (userId: string | null) => {
+  return useQuery<IAnalysisUser, CLErrors, IAnalysisUser, AnalysisUsersKeys>({
+    queryKey: usersKeys.item({ id: userId }),
+    queryFn: () => {
+      throw new Error(
+        'useAnalysisUserById query should never execute, there is no endpoint for this type of user. Only included resources'
+      );
```

Comment:
It's mostly about this pattern, and whether that seems acceptable to you. I'm doubting, but ultimately I don't think it's worth it to bloat to API more with another user endpoint, so I think this should work, and the hard error protect us.

Reply:
I understand the need to do this but I would suggest an alternative (depending on the effort). How hard would it be to not use the included resources for this but add author to the attributes of the input. In this way we include this (limited) data in the endpoint itself. The idea of the `included` resources is partly that we don't want to put all of this data in the endpoint but in this case, since we need only a fraction of the `user`, I think it's acceptable.